### DISCUSSION TITLE
Allow Directus Cloud to change QUERY_LIMIT_DEFAULT

### DISCUSSION THREAD

**Discussion by JackMullinkosson** [directus/directus#22179] [Feature Requests]
### Summary

Directus cloud users are not able to relate more than 100 items to a single item because of the QUERY_LIMIT_DEFAULT being set to 100. In the settings panel, users should have an option to change that. This feature would solve the issue described here: https://github.com/directus/directus/issues/22144

The potential solution suggested in the above issue is not viable because it does not account for users wishing to achieve this functionality through the app rather than through custom queries. I also reached out to cloud support to request they make the default limit higher for my specific use case but they denied my request.

### Motivation

Use case: You have a single venue related to hundreds of reservations. Or a single vendor related to hundreds of customers. In your database, you want to be able to see all of those customers or reservations and sort by the venue or vendors they are related to. You would also use this in Directus insights for example if you wanted to look through this type of data.

This is achievable if you are importing your data through custom queries, but the use case in which you are using the app, either through manual data entry or Directus flows, should be accounted for. 

### Detailed Design

Inside settings, a number input to allow users to change what their QUERY_LIMIT_DEFAULT would be. 

### Must Have:
- Option to relate more than 100 items to a single item

### Should Have:
- Ability to manually configure the QUERY_LIMIT_DEFAULT

### Could Have:
- A number format

### Drawbacks

Will add another field to the settings panel which will end up making it harder to find what setting you're looking for

### Alternatives

Perhaps it could be set on the collection itself

### Adoption Strategy

Is already achievable through queries and self hosted so wouldn't break anything

**Comment by ComfortablyCoding** [comment]
Per discussion on discord the `QUERY_LIMIT_DEFAULT` does not affect this, you can relate as many items as you wish on cloud and self host irrespective of this value. The cause for the described issue was due to an incorrect flow.

**Comment by JackMullinkosson** [comment]
For further information on what was incorrect about the flow that led to this issue, see this [discussion](https://github.com/directus/directus/issues/22212).
